tests/pages/api/apiUtils/index.test.ts

```diff
@@ -57,54 +55,49 @@ describe('apiUtils', () => {
     });
 
     describe('redirectOnFareType', () => {
-        it('should return 302 redirect to /periodType when period is selected', () => {
-            const { res } = getMockRequestAndResponse({}, null, {}, writeHeadMock);
-            const fareType = 'period';
-            redirectOnFareType(fareType, res);
-
-            expect(writeHeadMock).toBeCalledWith(302, { Location: '/periodType' });
+        it('should return 302 redirect to /service when the single ticket option is selected', () => {
+            const writeHeadMock = jest.fn();
```

Comment:
Was it not working with the `writeHeadMock` outside the tests and resetting after each?

Reply:
I'll be honest, I can't remember why I changed this. Have changed it back as it works perfectly fine as it was